app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -45,6 +45,8 @@ def save_ems_cloud_inventory(ems, hashes, target = nil, disconnect = true)
       _log.debug("#{log_header} hashes:\n#{YAML.dump(hashes)}")
     end
 
+    hashes[:tag_mapper].find_or_create_tags if hashes[:tag_mapper]
```

Comment:
Is the if condition ok? Seems like this would never run for Cloud?

Reply:
I see, that is set in AWS repo. So jut a question, why is  ```if hashes[:tag_mapper]``` and not in SaveInventoryContainer

Reply:
IINM it will be set by https://github.com/ManageIQ/manageiq-providers-amazon/pull/316 (but not other cloud providers, hence condition).

But I just run amazon specs assuming they prove tag mapping works — if as you say they're commented out, then I didn't really test it...

Reply:
ok, make sense, my brain must be still rebooting after the weekend :-)